test/intern/all.js

```diff
@@ -1,28 +1,28 @@
 define([
-	'intern/node_modules/dojo/has!host-browser?./core/createDestroy',
-	'intern/node_modules/dojo/has!host-browser?./core/addCssRule',
-	'intern/node_modules/dojo/has!host-browser?./core/setClass',
-	'intern/node_modules/dojo/has!host-browser?./core/columns',
-	'intern/node_modules/dojo/has!host-browser?./core/List',
-	'intern/node_modules/dojo/has!host-browser?./core/stores',
-	'intern/node_modules/dojo/has!host-browser?./core/_StoreMixin',
-	'intern/node_modules/dojo/has!host-browser?./core/OnDemandList',
-	'intern/node_modules/dojo/has!host-browser?./core/trackable',
-	'intern/node_modules/dojo/has!host-browser?./extensions/_insertNoDataNode',
-	'intern/node_modules/dojo/has!host-browser?./extensions/ColumnHider',
-	'intern/node_modules/dojo/has!host-browser?./extensions/ColumnResizer',
-	'intern/node_modules/dojo/has!host-browser?./extensions/CompoundColumns',
-	'intern/node_modules/dojo/has!host-browser?./extensions/DijitRegistry',
-	'intern/node_modules/dojo/has!host-browser?./extensions/Pagination',
-	'intern/node_modules/dojo/has!host-browser?./extensions/SingleQuery',
-	'intern/node_modules/dojo/has!host-browser?./mixins/ColumnSet',
-	'intern/node_modules/dojo/has!host-browser?./mixins/Editor',
-	'intern/node_modules/dojo/has!host-browser?./mixins/Editor-radio',
-	'intern/node_modules/dojo/has!host-browser?./mixins/Keyboard',
-	'intern/node_modules/dojo/has!host-browser?./mixins/Selection',
-	'intern/node_modules/dojo/has!host-browser?./mixins/Selector',
-	'intern/node_modules/dojo/has!host-browser?./mixins/Tree',
-	'intern/node_modules/dojo/has!host-browser?./mixins/Tree-expand-promise',
-	'intern/node_modules/dojo/has!host-browser?./mixins/Tree-additional-filter',
-	'intern/node_modules/dojo/has!host-browser?./mixins/Tree-indent'
+	'dojo/has!host-browser?./core/createDestroy',
```

Comment:
@jason0x43 is using `has!host-browser` still a recommended practice?

Reply:
That's fine if you're using the dojo loader. You can also handle that at the test config level with something like:

```json
{
  "suites": [ "tests/unit/suitesThatRunAnywhere/*" ],
  "browser": {
    "suites": ["tests/unit/suitesThatRunInBrowsers/*"]
  },
  "node": {
    "suites": ["tests/unit/suitesThatRunInNode/*"]
  }
}
```